### ISSUE TITLE
Installation instructions are incorrect

### ISSUE THREAD

**Issue by jkodumal** [liquidata-inc/dolt#106]
The suggested installation commands probably aren't correct:

```
$ tar -xf /your/download/location/dolt-darwin-amd64.tar.gz -C /usr/local/bin
$ ln -s /usr/local/lib/dolt /usr/local/lib/dolt/bin/dolt
```

The fist command creates the directory `/usr/local/bin/dolt-darwin-amd64`. Given that, the second command doesn't make sense.

**Comment by oscarbatori** [comment]
@jkodumal thanks, good catch. We fixed it.

**Comment by mjesuele** [comment]
Fixed by #108
